src/hmrc-content-style-guide/index.njk

```diff
@@ -147,20 +147,22 @@ return’ if possible.</p>
         content: {
           html: "
 <h3 class='govuk-heading-s' id='dashes'>dashes</h3>
-<p class='govuk-body'>Dashes separate words into clauses. They are slightly longer than a hyphen and always
-have spaces either side of them. For example ‘You can claim the actual amount you’ve spent – you’ll need
-to keep receipts.’</p>
-<p class='govuk-body'>Dashes usually make sentences more complicated. Before using a dash, or pair of
-dashes, consider using separate sentences.</p>
-<p class='govuk-body'>There are different ways to include a dash in content. You can:</p>
+<p class='govuk-body'>Dashes separate groups of words into clauses. They are longer than a hyphen and always
+have spaces either side of them. For example ‘You can claim the actual amount you’ve spent &mdash; you’ll need to keep receipts.’</p>
+
+<p class='govuk-body'>Dashes usually make sentences more complicated. Before using a dash, or pair of dashes, consider using separate sentences.</p>
+
+<p class='govuk-body'>There are different ways to include an em dash in content. You can:</p>
+
 <ul class='govuk-list govuk-list--bullet'>
-  <li>use option + - key on an Apple Mac</li>
-  <li>use alt + 0150 on a PC with a number pad</li>
-  <li>copy – into your content</li>
+  <li>use option + shift + minus key on an Apple Mac</li>
+  <li>use alt + 0151 on a PC</li>
```

Comment:
This has to be on the number pad so worth pointing out

Reply:
Ah, ok. I was surprised by that. Have updated.